package/yast2-network.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Mon Apr  4 17:24:13 UTC 2016 - mfilka@suse.com
+
+- bnc#955217, bnc#956605
+  - the fix was originaly enabled only for ssh / vnc installation
+    since now is udev handling completely and always handled in
+    the first stage
```

Comment:
So there is no change for the user, right? I think description is easier to understand:

> Unified udev rule handling by always doing it in the 1st stage of installation.